### ISSUE TITLE
[Bug] Inaccurately Reporting OS Drive Size

### ISSUE THREAD

**Issue by lhoggatt17** [MauriceNino/dashdot#369]
### Description of the bug

This app is awesome by the way! This was exactly what I was looking for to monitor my system, especially my disk space. 

Unfortunately I'm having an issue with the system reporting the incorrect usage for my OS disk. It is reporting my two HDDs correctly. Here is what happens when I hover over the top (`/`) disk:
![image](https://user-images.githubusercontent.com/37279082/189799287-4e9e9efb-dee2-4851-94a2-f99f0736b318.png)

Here is the output of `df -h --total`:
```
Filesystem                         Size  Used Avail Use% Mounted on
tmpfs                              1.6G  5.7M  1.6G   1% /run
/dev/mapper/ubuntu--vg-ubuntu--lv  914G   62G  814G   8% /
tmpfs                              7.8G     0  7.8G   0% /dev/shm
tmpfs                              5.0M     0  5.0M   0% /run/lock
/dev/sda2                          2.0G  248M  1.6G  14% /boot
/dev/sda1                          1.1G  5.3M  1.1G   1% /boot/efi
/dev/sdb2                          2.8T  1.7T  1.2T  59% /mnt/storage
tmpfs                              1.6G  4.0K  1.6G   1% /run/user/1000
/dev/sdc1                          2.7T  1.5T  1.2T  56% /mnt/localBackup
total                              6.4T  3.1T  3.1T  51% -
```

I believe I should be seeing 62G (what is mounted on `/`) rather than 252M. You can see down in the 'relevant log' section that the docker container is getting this value somehow. 

I'm not sure what is going on, but I'm happy to run any commands and give you more info if you need it. 

### How to reproduce

I installed ubuntu server 22.04 using the standard process, so I imagine this will happen for other people running stock Ubuntu Server 22.04. 

### Relevant log output

```shell
dashdot-dash-1  | First measurement [Storage]: {
dashdot-dash-1  |   layout: [
dashdot-dash-1  |     { load: 264990720 },
dashdot-dash-1  |     { load: 1761617133568 },
dashdot-dash-1  |     { load: 1551355965440 }
dashdot-dash-1  |   ]
dashdot-dash-1  | }
```

### Info output of dashdot cli

```shell
yarn run v1.22.19
$ node dist/apps/cli/main.js info
INFO
=========
Yarn: 1.22.19
Node: v18.8.0
Dash: 4.4.0

Cwd: /app
Hash: 78aea0f9ae2c711698c3b947c8591579b366ad85
In Docker: true
In Podman: false
In Docker (env): true
Done in 0.50s.
```

### What browsers are you seeing the problem on?

Chrome

### Where is your instance running?

Linux Server

**Comment by MauriceNino** [comment]
Hey there, thanks for creating an issue!
I hope that I can look into it real soon - in the meantime, it would help me a lot if you could run the following command and paste the output in a comment:

```
docker exec CONTAINER yarn cli raw-data --storage
```

**Comment by lhoggatt17** [comment]
Here you go:

``` 
Disk Layout: [
  {
    device: '/dev/sda',
    type: 'SSD',
    name: 'CT1000MX500SSD1 ',
    vendor: 'ATA',
    size: 1000204886016,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '043',
    serialNum: '',
    interfaceType: 'SATA',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/sdb',
    type: 'HD',
    name: 'ST3000DM001-1CH1',
    vendor: 'Seagate',
    size: 3000592982016,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: 'CC26',
    serialNum: '',
    interfaceType: 'SATA',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/sdc',
    type: 'HD',
    name: 'TOSHIBA DT01ACA3',
    vendor: 'Toshiba',
    size: 3000592982016,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: 'ABB0',
    serialNum: '',
    interfaceType: 'SATA',
    smartStatus: 'unknown',
    temperature: null
  }
]
FS Size: [
  {
    fs: 'overlay',
    type: 'overlay',
    size: 980666998784,
    used: 66848079872,
    available: 872830881792,
    use: 7.11,
    mount: '/'
  },
  {
    fs: '/dev/mapper/ubuntu--vg-ubuntu--lv',
    type: 'ext4',
    size: 980666998784,
    used: 66848079872,
    available: 872830881792,
    use: 7.11,
    mount: '/mnt/host'
  },
  {
    fs: '/dev/loop0',
    type: 'squashfs',
    size: 65011712,
    used: 65011712,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/core20/1611'
  },
  {
    fs: '/dev/loop1',
    type: 'squashfs',
    size: 66322432,
    used: 66322432,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/core20/1623'
  },
  {
    fs: '/dev/loop2',
    type: 'squashfs',
    size: 83886080,
    used: 83886080,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/lxd/22923'
  },
  {
    fs: '/dev/loop3',
    type: 'squashfs',
    size: 108003328,
    used: 108003328,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/lxd/23541'
  },
  {
    fs: '/dev/loop4',
    type: 'squashfs',
    size: 49283072,
    used: 49283072,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/snapd/16292'
  },
  {
    fs: '/dev/loop5',
    type: 'squashfs',
    size: 50331648,
    used: 50331648,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/snapd/16778'
  },
  {
    fs: '/dev/sda2',
    type: 'ext4',
    size: 2040373248,
    used: 259497984,
    available: 1656725504,
    use: 13.54,
    mount: '/mnt/host/boot'
  },
  {
    fs: '/dev/sda1',
    type: 'vfat',
    size: 1124995072,
    used: 5492736,
    available: 1119502336,
    use: 0.49,
    mount: '/mnt/host/boot/efi'
  },
  {
    fs: '/dev/sdb2',
    type: 'fuseblk',
    size: 3000457228288,
    used: 1860892782592,
    available: 1139564445696,
    use: 62.02,
    mount: '/mnt/host/mnt/storage'
  },
  {
    fs: '/dev/sdc1',
    type: 'ext4',
    size: 2952324628480,
    used: 2095100137472,
    available: 707178143744,
    use: 74.76,
    mount: '/mnt/host/mnt/localBackup'
  }
]
Block Devices: [
  {
    name: 'sda',
    type: 'disk',
    fsType: '',
    mount: '',
    size: 1000204886016,
    physical: 'SSD',
    uuid: '',
    label: '',
    model: 'CT1000MX500SSD1 ',
    serial: '',
    removable: false,
    protocol: 'sata',
    group: undefined
  },
  {
    name: 'sdb',
    type: 'disk',
    fsType: '',
    mount: '',
    size: 3000592982016,
    physical: 'HDD',
    uuid: '',
    label: '',
    model: 'ST3000DM001-1CH1',
    serial: '',
    removable: false,
    protocol: 'sata',
    group: undefined
  },
  {
    name: 'sdc',
    type: 'disk',
    fsType: '',
    mount: '',
    size: 3000592982016,
    physical: 'HDD',
    uuid: '',
    label: '',
    model: 'TOSHIBA DT01ACA3',
    serial: '',
    removable: false,
    protocol: 'sata',
    group: undefined
  },
  {
    name: 'loop0',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/core20/1611',
    size: 64966656,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'loop1',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/core20/1623',
    size: 66293760,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'loop2',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/lxd/22923',
    size: 83832832,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'loop3',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/lxd/23541',
    size: 107986944,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'loop4',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/snapd/16292',
    size: 49242112,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'loop5',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/snapd/16778',
    size: 50323456,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'sda1',
    type: 'part',
    fsType: 'vfat',
    mount: '/mnt/host/boot/efi',
    size: 1127219200,
    physical: '',
    uuid: '6FB0-B860',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'sda2',
    type: 'part',
    fsType: 'ext4',
    mount: '/mnt/host/boot',
    size: 2147483648,
    physical: '',
    uuid: '2b85179e-13b0-4c68-9c8f-fa9fd260c45b',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'sda3',
    type: 'part',
    fsType: 'LVM2_member',
    mount: '',
    size: 996927340544,
    physical: '',
    uuid: 'bnl9js-ZjYl-thvJ-g1aX-CPRQ-sRU0-bNZOnr',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'sdb1',
    type: 'part',
    fsType: '',
    mount: '',
    size: 134217728,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'sdb2',
    type: 'part',
    fsType: 'ntfs',
    mount: '/mnt/host/mnt/storage',
    size: 3000457232384,
    physical: '',
    uuid: 'BA02C95802C919F1',
    label: 'Backups+',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'sdc1',
    type: 'part',
    fsType: 'ext4',
    mount: '/mnt/host/mnt/localBackup',
    size: 3000591450112,
    physical: '',
    uuid: '30cbdafb-2355-4ad2-957d-1c1579e6b82d',
    label: 'LocalBackup',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  }
]

```

**Comment by MauriceNino** [comment]
:tada: This issue has been resolved in version 4.5.0

Please check the [changelog](https://github.com/MauriceNino/dashdot/blob/main/.github/CHANGELOG.md) for more details.

**Comment by hoggatt** [comment]
Thanks so much! This looks awesome:
![image](https://user-images.githubusercontent.com/37279082/194598508-ba8ad589-949d-48d4-b431-685fd418e80d.png)
